tests/bundle/test_bundle.py

```diff
@@ -11,38 +11,239 @@
 )
 
 ALLOWED_OPS_PER_UNSTAKED_SENDER = 4
+DEFAULT_MAX_PRIORITY_FEE_PER_GAS = 10**9
+DEFAULT_MAX_FEE_PER_GAS = 5*10**9
+MIN_PRICE_BUMP = 10
 
+def bump_fee_by(fee, factor):
+    return round((fee*(100+factor)/100))
+
+
+@pytest.mark.parametrize("mode", ["manual"], ids=[""])
+@pytest.mark.usefixtures("clear_state", "set_bundling_mode")
+def test_bundle_replace_with_only_priority_fee_bump(w3):
```

Comment:
These tests seems ok, but it is very hard to read and verify, as it is a copy-and-paste of a lot of information (with minor modifications)

Most tests are basically:
`test_replace_fee(bump_prio, bump_max, outcome)`
where "outcome" is a function that gets 2 ops, expects the 1st to succeed and the 2nd is either replace or fail.

the above test function can then be parameterized by adding before it:
```
@pytest.mark.parametrize("bump_prio,bump_max,outcome",[
  [0,0,fail_replace],
  [10,0,fail_replace],
...
  [10,10,succeed_replace]
])
```

Reply:
Agree that parameterising the test would be a lot better for readability. I will refactor in the next few days.